Assets/Scripts/GameScreen.cs

```diff
@@ -0,0 +1,64 @@
+﻿#region Copy
+// This file is © 2022 Rumyancev Pavel <paulrumyancev@gmail.com>
+// Vector. School of Game development
+#endregion
+using UnityEngine;
+using VectorSchool.DataLayer;
+
+namespace VectorSchool.VisualLayer
```

Comment:
Please make sure your namespce  structure match the folders structure. This script should be under "VisualLayer" folder then